### PR TITLE
NC-5129, NC-5130: Documentation update for Turnstile (flex-width, size, feedback toggle)

### PR DESCRIPTION
**Pull request by worenga**

### CODE REVIEW

File: content/turnstile/get-started/client-side-rendering.md [hunk 1]
```diff
@@ -243,9 +244,10 @@ The Turnstile widget can have two different sizes when using the Managed or Non-
...
 | `refresh-expired` | `data-refresh-expired` | Automatically refreshes the token when it expires. Can take `auto`, `manual`, or `never`, defaults to `auto`. |
 | `refresh-timeout` | `data-refresh-timeout` | Controls whether the widget should automatically refresh upon entering an interactive challenge and observing a timeout. Can take `auto` (automatically refreshes upon encountering an interactive timeout), `manual` (prompts the visitor to manually refresh) or `never` (will show a timeout), defaults to `auto`.  Only applies to widgets of mode managed.|
 | `appearance` | `data-appearance` | Appearance controls when the widget is visible. It can be `always` (default), `execute`, or `interaction-only`. Refer to [Appearance modes](/turnstile/get-started/client-side-rendering/#appearance-modes) for more information. |
+| `feedback-enabled` | `data-feedback-enabled` | Allow's Cloudflare to gather visitor feedback upon widget failure. It can be `true` (default) or `false` |
```
Comment (migueldemoura) [inline content/turnstile/get-started/client-side-rendering.md]: Allows

File: data/changelogs/turnstile.yaml [hunk 2]
```diff
@@ -5,6 +5,11 @@ productLink: "/turnstile/"
 productArea: Application security
 productAreaLink: /fundamentals/reference/changelog/security/
 entries:
+- publish_date: '2024-08-12'
+  description: |-
+    - Added [`[flexible]`](/turnstile/get-started/client-side-rendering/#Widget-size width.
+    - New dimensions for Turnstile compact size.
+    - Feedback Report Toggle on Widget Configuration
```
Comment (migueldemoura) [inline data/changelogs/turnstile.yaml]: Add period at the end

**Review by migueldemoura — CHANGES_REQUESTED** [review]

File: data/changelogs/turnstile.yaml [hunk 3]
```diff
@@ -5,6 +5,11 @@ productLink: "/turnstile/"
 productArea: Application security
 productAreaLink: /fundamentals/reference/changelog/security/
 entries:
+- publish_date: '2024-08-12'
+  description: |-
+    - Added [`[flexible]`](/turnstile/get-started/client-side-rendering/#Widget-size width.
+    - New dimensions for Turnstile compact size.
+    - Feedback Report toggle on Widget's configuration.
```
Comment (patriciasantaana) [inline data/changelogs/turnstile.yaml]:
```suggestion
- publish_date: '2024-08-12'
  description: |-
    - Added [`[flexible]`](/turnstile/get-started/client-side-rendering/#widget-size) width widget size.
    - Added new dimensions for Turnstile's compact size.
    - Added a Feedback Report toggle on the widget's configuration.
```

**Review by patriciasantaana — APPROVED** [review]

**Review by migueldemoura — APPROVED** [review]

**Comment by patriciasantaana** [comment]
@worenga closing this PR in favor of #16108 due to errors from the migration resulting in the need to remake it.

**Pull request closed without merging**